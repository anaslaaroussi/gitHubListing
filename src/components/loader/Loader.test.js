import { shallow } from "enzyme";
import React from "react";
import { Loader } from "./Loader";

it("expect render loader component", () => {
  expect(shallow(<Loader />)).toMatchSnapshot();
});
