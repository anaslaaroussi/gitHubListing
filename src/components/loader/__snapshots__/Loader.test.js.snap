// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect render loader component 1`] = `ShallowWrapper {}`;
